### ISSUE TITLE
Runner에서 load 후에도 train이 가능하게 수정 해야함

### ISSUE THREAD

**Issue by kuro11pow2** [kuro11pow2/graduation-project#4]
지금은 재학습을 고려하지 않은 코드임.

모델로 추론하려면 model.eval() 을 호출하여 dropout과 batch normalization layer를 evaluation 모드로 바꿈.
eval이 끝난 후 이어서 학습하고 싶다면 꼭 model.train()를 호출해야 함.

https://bluehorn07.github.io/2021/02/27/model-eval-and-train.html
